Header header
string name
lthmi_nav/IntMap mapa
geometry_msgs/Pose init_pose
